.storybook/main.js

```diff
@@ -15,6 +22,9 @@ module.exports = {
       },
     },
   ],
+    features: {
+    previewMdx2: true,
+  },
```

Comment:
Out of curiosity, why was this required?

Reply:
It is not! It was leftover from experimenting while trying to get storybook to work and forgot, removed now. Thanks!!